File: opengallery/src/main/java/com/lotus/opengallery/utils/ToastUtil.java
package com.lotus.opengallery.utils;

import android.content.Context;
import android.widget.Toast;

public class ToastUtil {
    public static void showToast(Context context,String msg) {
        Toast.makeText(context,msg,Toast.LENGTH_SHORT).show();
    }

    public static void shortShow(Context context,String msg) {
       Toast.makeText(context,msg,Toast.LENGTH_SHORT).show();
    }

    public static void longShow(Context context,String msg) {
        Toast.makeText(context,msg,Toast.LENGTH_LONG).show();
    }
}

File: app/src/main/java/com/lotus/ourhome/utils/ToastUtil.kt
package com.lotus.ourhome.utils

import android.content.Context
import android.widget.Toast
import com.lotus.ourhome.base.App

object ToastUtil {
    fun showToast(context: Context,msg:String){
        Toast.makeText(context,msg,Toast.LENGTH_SHORT).show();
    }

    fun showToast(msg:String){
        Toast.makeText(App.instance,msg,Toast.LENGTH_SHORT).show();
    }
}